tests/interlocking_component/test_route_controller.py

```diff
@@ -1,17 +1,27 @@
 import os
+from uuid import uuid1
```

Comment:
What is this import for?

Hint: All our uuids are generated using `uuid4`.

Reply:
I don't even remember why I put it there, so I removed it.

Reply:
Ok, happens :D